python/cudf/cudf/utils/ioutils.py

```diff
@@ -771,6 +771,33 @@
 0  123 2018-11-13T12:00:00.000 5451
 1  456 2018-11-14T12:35:01.000 5784
 2  789 2018-11-15T18:02:59.000 6117
+"""
+
+"""
```

Comment:
We shouldn't be breaking this string here I believe, we also don't need to redefine the `Examples` section